src/System.Net.Requests/tests/System.Net.Requests.Tests.builds

```diff
@@ -2,14 +2,15 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
+    <Project Include="System.Net.Requests.Tests.csproj"/>
     <Project Include="System.Net.Requests.Tests.csproj">
-      <OSGroup>Unix</OSGroup>
```

Comment:
I believe so.  I don't see anything Unix-specific in the test project.  Searching for "Unix" case-insensitive in all files in the tests directory yields only the two lines for configuring the solution selection.